doc/qtcreator/src/mer/creator-developing-mer.qdoc

```diff
@@ -24,175 +24,83 @@
-            the kit that you specified above.
-
-    \endlist
-
-    \section1 Specifying Run Settings for Sailfish OS Devices
-
-    To specify run settings, select \gui Projects > \gui {Build & Run} >
-    \gui Run. Usually, you can use the default settings.
-
-    When you run the project, \QC deploys the application as specified by the
-    deploy steps. By default, \QC copies the binary files to the connected
-    device or to the emulator. For more information, see
-    \l{Deploying Applications to Sailfish OS Devices}.
-
-    \section1 Adding Sailfish OS Build Targets
-
-    To add build targets, switch to \gui {Sailfish OS} mode to access the
-    build engine's \gui {Control Centre}:
-
-    \image creator-mer-sdk-control-center.png
-
-    \list 1
-
-        \li Select the \gui Targets tab.
-
-        \li In the \gui Toolchain field, select a tool chain.
-
-        \li In the \gui {Target name} field, give a name to the target. \QC
-            copies the necessary information to a folder that is named after the
-            target.
-
-        \li In the \gui {Target URL} field, select the URL to download the
-            sysroot image. For example,
-            \c{http://img.merproject.org/images/web/lbt/2-20130522-204538/mer-sdk-i486-chroot-full.tar.bz2}
-
-        \li Select \gui Add to add the target.
-
-    \endlist
-
-    \section1 Configuring Connections
-
-    To configure a connection between \QC and a Sailfish OS build engine running
-    in a virtual machine:
-
-    \list 1
+    \section1 Adding Sailfish OS Devices
 
-        \li Select \gui Tools > \gui Options > \gui {Sailfish OS} > \gui {Build
-            Engine} > \gui {Add...}.
+    Start by establishing network connection between your device and the
+    development PC. One option is to connect the device with an USB cable and
+    switch the USB connection mode on the device to Developer Mode. Another
+    option is to access the device through a local Wi-Fi network.
 
-            \image creator-mer-options-vm.png
+    Open the dialog for adding the device by selecting \gui {Tools > Options >
+    Devices > Add... > Sailfish OS Device}.
 
-            \note You can create only one connection to a particular build engine.
+    \image creator-sailfish-device-connection.png
 
-        \li Deselect the \gui {No window} check box and restart the virtual machine
-            to show the VirtualBox user interface.
+    On the \gui Connection page, fill in the device hostname or IP address.
+    Device's IP address may be retrieved under the \gui Settings application,
+    under the \gui {Developer tools} category.
 
-        \li In the \gui Connection field, specify the SSH key to use to protect
-            the connection between \QC and the build engine:
+    The traditional device username was \c nemo. In recent Sailfish OS version
+    this changed to \c defaultuser. You may need to adjust the \gui Username
+    field to match your device username.
```

Comment:
It might be worth mentioning that the username is visible in the same view as the IP address on the device

Reply:
Did some reorganization